stdlib/public/runtime/SwiftRT-COFF.cpp

```diff
@@ -64,16 +67,18 @@ static void swift_image_constructor() {
       nullptr,
       nullptr,
 
-      SWIFT_SECTION_RANGE(sw5prt),
-      SWIFT_SECTION_RANGE(sw5prtc),
-      SWIFT_SECTION_RANGE(sw5tymd),
-
-      SWIFT_SECTION_RANGE(sw5tyrf),
-      SWIFT_SECTION_RANGE(sw5rfst),
-      SWIFT_SECTION_RANGE(sw5flmd),
-      SWIFT_SECTION_RANGE(sw5asty),
-      SWIFT_SECTION_RANGE(sw5repl),
-      SWIFT_SECTION_RANGE(sw5reps),
+      SWIFT_SECTION_RANGE(swift5_protocols),
+      SWIFT_SECTION_RANGE(swift5_protocol_conformances),
+      SWIFT_SECTION_RANGE(swift5_type_metadata),
+
+      SWIFT_SECTION_RANGE(swift5_typeref),
+      SWIFT_SECTION_RANGE(swift5_reflstr),
+      SWIFT_SECTION_RANGE(swift5_fieldmd),
+      SWIFT_SECTION_RANGE(swift5_assocty),
+      SWIFT_SECTION_RANGE(swift5_replace),
+      SWIFT_SECTION_RANGE(swift5_replac2),
+      SWIFT_SECTION_RANGE(swift5_builtin),
+      SWIFT_SECTION_RANGE(swift5_capture),
```

Comment:
The section names should rename the same, just the names for the structure are the same.